content/tutorials/getting-started.md

```diff
@@ -235,7 +247,7 @@ This starts a KES server on `127.0.0.1:7373` and stores keys in memory.
 To upgrade KES, follow the getting started steps and replace the KES binary with the newer version on each KES server node.
 
 {{< admonition type="important" >}}
-You cannot revert to a previous version of KES after upgrading.
+After upgrading to release `2024-02-29T08-12-28Z` or later, you cannot revert to a release prior to `2024-02-29T08-12-28Z` due to changes in how KES processes ciphertext.
```

Comment:
Maybe?
```suggestion
Due to changes in how KES processes ciphertext, it is not possible to revert to an earlier version after upgrading to release `2024-02-29T08-12-28Z` or later.
```
 I guess we could include a "test in a lower environment" mention also.

Reply:
This is much, much better. Thanks!
Added a lower environment sentence, too.